dlib/cmake_utils/dlibConfig.cmake.in

```diff
@@ -23,20 +23,20 @@
 get_filename_component(dlib_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
 
 if(${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} VERSION_LESS 2.8)
-    get_filename_component(dlib_INSTALL_PATH "@CONF_INSTALL_PATH@" ABSOLUTE)
+    get_filename_component(dlib_INSTALL_PATH "@CMAKE_INSTALL_PREFIX@" ABSOLUTE)
```

Comment:
Do we still need these get_filename_component lines at all if we switch to this method?